usr/share/rear/output/RAWDISK/Linux-i386/270_create_grub2_efi_bootloader.sh

```diff
@@ -5,7 +5,17 @@
 ### Check prerequisites
 
 # Run only if no EFI bootloader has been created yet and Grub 2 EFI is available
-([[ -n "$RAWDISK_BOOT_EFI_STAGING_ROOT" ]] || ! has_binary grub-mkimage || ! [[ -d /usr/lib/grub/x86_64-efi ]]) && return 0
+
+grub_name="grub2"
+if ! test -d "/boot/$grub_name" ; then
+    grub_name="grub"
+    if ! test -d "/boot/$grub_name" ; then
+        LogPrintError "Cannot install GRUB2 (neither boot/grub nor boot/grub2 directory)"
```

Comment:
In contrast to
usr/share/rear/finalize/Linux-i386/660_install_grub2.sh
where we do not error out, see the reason behind in
https://github.com/rear/rear/blob/master/usr/share/rear/finalize/Linux-i386/660_install_grub2.sh#L34
which is an example for "know the rules so you know how to break them properly"
cf.  https://github.com/rear/rear/wiki/Coding-Style
in this case here it should error out according to
"In case of errors better abort than to blindly proceed"
in https://github.com/rear/rear/wiki/Coding-Style
because the `output` stage (i.e. scripts below usr/share/rear/output/)
are run during `rear mkrescue/mkbackup` and here is is important
to error out when things fail so that it is obvious for the user
that later `rear recover` won't work for him, cf.
https://github.com/rear/rear/wiki/Developers-Guide

Also the error message should be adapted to match
what the comments tell what this script does, e.g. something like
```
Error "Cannot use GRUB2 to create EFI bootloader (neither boot/grub nor boot/grub2 directory in $TARGET_FS_ROOT)"
```

Reply:
@OliverO2 
thank you for your review!
A perfect example how the review from the actual code owner helps
in contrast to a review from someone else (like me).

Regarding (1.):
Should there be any user visible message at this place
or should it silently `return 1` or perhaps even `return 0`
because at this place there is not yet a real error?

Regarding (2.):
The `grub_name` variable name is because it is also used in
usr/share/rear/finalize/Linux-i386/660_install_grub2.sh
cf. https://github.com/rear/rear/issues/2275#issuecomment-553923647
so it is also used here by copy-and-paste to implement here
the same kind of functionality.